privacy_evaluator/attacks/membership_inference/on_point_basis.py

```diff
@@ -0,0 +1,107 @@
+from .membership_inference import MembershipInferenceAttack
+from ...classifiers.classifier import Classifier
+import numpy as np
+from typing import Tuple
+
+
+class MembershipInferenceAttackOnPointBasis(MembershipInferenceAttack):
+    """MembershipInferenceBlackBoxAttack class."""
+
+    def __init__(
+        self,
+        target_model: Classifier,
+        x_train: np.ndarray,
+        y_train: np.ndarray,
+        x_test: np.ndarray,
+        y_test: np.ndarray,
+    ):
+        """Initializes a MembershipInferenceOnPointBasis class.
+
+        :param target_model: Target model to be attacked.
+        :param x_train: Data which was used to train the target model.
+        :param y_train: True, one-hot encoded labels for `x_train`.
```

Comment:
What do you mean by true here? Isn't it a vector with the labels only and not a Boolean value?